### ISSUE TITLE
Yii 2.2 Release

### ISSUE THREAD

**Issue by rhertogh** [yiisoft/yii2#20179]
### First Release
This will mark a first milestone in efforts to push Yii2 to a  more modern framework for those who want to stay on Yii without being left behind.  As discussed [here](https://github.com/yiisoft/yii2/discussions/20138) Yii 2.0.x will be "bug fixes and security fixes only" from now on, so new features should target the 2.2 branch.

For more discussion on 2.2 see [Yii 2.2 Discussions](https://github.com/yiisoft/yii2/discussions/19831).

---
Tasks for the first release
- [x] Create 22 branch
- [ ]  Update README to Provide summary of 2.2 and how it differs from classic Yii2
- [ ] Make JQuery Optional to allow for (#20528)
- [ ] Make Components Resolution prefer DI over components config (#20574)
- [ ] Extra branches created in all extensions.
- [ ] Extension PRs merged into master.
- [ ] Everything tested together.
- [ ] Update release info on [website](https://www.yiiframework.com/release-cycle) and [readme.md](https://github.com/yiisoft/yii2/blob/master/README.md)
- [ ]  Release for framework and extensions

**Comment by samdark** [comment]
We need extensions:

1. Extra branches created in all extensions.
2. Extension PRs merged into master.
3. Everything tested together.

**Comment by schmunk42** [comment]
This https://github.com/yiisoft/yii2/discussions/20164#discussioncomment-9906412 seems promising.

**Comment by mtangoo** [comment]
> This [#20164 (reply in thread)](https://github.com/yiisoft/yii2/discussions/20164#discussioncomment-9906412) seems promising.

Yes it is. We just need to do actual testing and see what happens

**Comment by mtangoo** [comment]
@schmunk42  @rhertogh @terabytesoftw @samdark your opinions

**Comment by terabytesoftw** [comment]
I'm going to close this and open a new one, with the current status and what's next to be done.
